kite-tools-parent/kite-tools/src/test/java/org/kitesdk/cli/commands/TestCSVSchemaCommand.java

```diff
@@ -109,6 +116,17 @@ public void testSchemaRequiredFields() throws Exception {
   }
 
   @Test
+  public void testSchemaSpecifiedFields() throws Exception {
```

Comment:
This will require tests for:
* String data when int is the specified type
* Float data when int is the specified type
* String data when float is the specified type
* Extra specified types (what happens? ignored?)